integration_testing/features/learner/learner-engage-content-recommended.feature

```diff
@@ -7,20 +7,20 @@ Feature: Learner engages recommended content
       And I am on *Learn > Recommended* page
 
   Scenario: Engage with content on Recommended tab
-    When I see content items under the *Most popular*, *Next steps* and *Resume* headings
+    When I see content items under the *Most popular* and *Recommended* headings
+    # *Next steps* and *Resume* ???
```

Comment:
@khangmach Please confirm that the *Next steps* and *Resume* sections were removed intentionally from the **Recommended** tab in 0.13.